orbit/pkg/useraction/mdm_migration_darwin.go

```diff
@@ -374,17 +416,24 @@ func (m *swiftDialogMDMMigrator) renderMigration() error {
 				}
 			}
 
+			if err := m.mrw.SetMigrationFile(constant.MDMMigrationTypeManual); err != nil {
+				log.Error().Err(err).Msg("set migration file")
```

Comment:
not stopping the whole migration makes sense, should we report the error to the server?